src/theory/quantifiers/theory_quantifiers.cpp

```diff
@@ -68,6 +70,11 @@ TheoryQuantifiers::TheoryQuantifiers(Context* c,
   // pointer will be retreived by TheoryEngine and set to all theories
   // post-construction.
   d_quantEngine = d_qengine.get();
+
+  if (options::macrosQuant())
+  {
+    d_qmacros.reset(new QuantifiersMacros(d_qreg));
```

Comment:
I might be completely off here, but shouldn't this `new` get `delete`ed somewhere?

Reply:
Its allocated into a unique_ptr, this is automatic.